packages/server/src/user/__tests__/user.service.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { JwtModule } from '@nestjs/jwt';
+import { Test } from '@nestjs/testing';
+import { PrismaService } from '../../prisma/prisma.service';
+
+describe('UserModule Int-Test', () => {
+  let prisma: PrismaService;
```

Comment:
For testing, a mock should be used so you aren't saving into a real database

Reply:
Do I still need to use a mock if I'm using a test DB?

Reply:
Typically for unit tests you want to have no dependencies and test only the "component under test". For E2E testing is where you bring in a test database. Thats because unit tests typically test only a single thing and should do so very quickly.